internal/testing/e2e/query_tickets_test.go

```diff
@@ -153,7 +153,10 @@ func returnedByQuery(t *testing.T, tc testcases.TestCase) (found bool) {
 	om := newOM(t)
 
 	{
-		resp, err := om.Frontend().CreateTicket(context.Background(), &pb.CreateTicketRequest{Ticket: tc.Ticket})
+		ticket := pb.Ticket{}
+		ticket.SearchFields = tc.SearchFields
```

Comment:
Nit:
```
ticket := &pb.Ticket{
  SearchFields: tc.SearchFields,
}
```
Here and below.

Prefer setting fields on struct during initialization like this if possible.

Reply:
Done